docker-compose.yml

```diff
@@ -5,8 +5,8 @@ services:
     build:
       context: .
       dockerfile: web.Dockerfile
-    environment:
-      REACT_APP_BACKEND_URL: "http://localhost:8063"
+      args: 
+        REACT_APP_BACKEND_URL: "http://localhost:8063"
```

Comment:
every time you change this value, please rebuild the image by `docker-compose up --build web`